Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.cpp

```diff
@@ -2367,7 +2367,6 @@ namespace AzToolsFramework
         optionV4.widget->style()->drawControl(QStyle::CE_ItemViewItem, &optionV4, painter);
 
         AzToolsFramework::RichTextHighlighter::PaintHighlightedRichText(entityNameRichText, painter, optionV4, textRect);
-        painter->restore();
```

Comment:
Looking at the top of this `PaintEntityNameAsRichText` method it actually does a `painter->save()` on line 2312, but then never restores it, so I think we actually should add one here